tests/common/ixia/ixia_helpers.py

```diff
@@ -0,0 +1,439 @@
+            retval.append(string)
+        retval.sort()
+        return(retval)
+
+    def get_fanout_device_details (self, device_number) :
+        """With the help of this function you can select the chassis you want
+        to access. For example get_fanout_device_details(0) selects the 
+        first chassis. It just select the chassis but does not return 
+        anything. The rest of  the function then used to extract chassis 
+        information like "get_chassis_ip()" will the return the ip address 
+        of chassis 0 - the first chassis in the list.
+
+        Note:
+            Counting or indexing starts from 0. That is 0 = 1st cassis, 
+            1 = 2nd chassis ...
+
+        Args:
+           device_number (int): the chassis index (0 is the first)
+
+        Returns:
+           None   
+        """
+
+        # Pointer to chassis info  
+        self.last_fanout_assessed = device_number
+
+        # Chassis connection details
+        self.last_device_connection_details = \
+            self.fanout_list[self.last_fanout_assessed]['device_conn']
+
+        # Chassis ip details
+        self.ip_address = \
+        self.fanout_list[self.last_fanout_assessed]['device_info']['mgmtip'] 
+
+        # List of chassis cards and ports 
+        self.current_ixia_port_list = \
+             self.__parse_fanout_connections__()
+
+        #return self.fanout_list[self.last_fanout_assessed]
+
+    def get_connection_details (self) :
+        """This function returns all the details associated with a particular
+        chassis (selected earlier using get_fanout_device_details() function).
+        Details of the chassis will be available like chassis IP, card, ports, 
+        peer port etc. in a dictionary format.
+
+        Note: If you have not used get_fanout_device_details(), by default 0th
+            (first) chassis remains selected.
+
+        Args:
+            This function takes no argument.
+
+        Returns:
+            Details of the chassis connection as dictionary format.
+        """
+        return(self.last_device_connection_details)
+  
+    def get_chassis_ip (self) :
+        """This function returns IP address of a particular chassis 
+        (selected earlier using get_fanout_device_details() function).
+
+        Note: If you have not used get_fanout_device_details(), by default 0th
+            (first) chassis remains selected.
+
+        Args:
+            This function takes no argument.
+
+        Returns:
+            The IP address 
+        """
+        return self.ip_address
+       
+    def get_ports(self) :
+        """This function returns list of ports associated with a chassis 
+        (selected earlier using get_fanout_device_details() function) 
+        as a list of dictionary.
+
+        Note: If you have not used get_fanout_device_details(), by default 0th
+            (first) chassis remains selected.
+
+        Args:
+            This function takes no argument.
+
+        Returns:
```

Comment:
Returns -> Return

Reply:
Google style docstring recommends “Returns” instead of “Return”(https://google.github.io/styleguide/pyguide.html). So, I kept it as it is.